sdks/python/apache_beam/io/aws/clients/s3/boto3_client.py

```diff
@@ -0,0 +1,239 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from __future__ import absolute_import
+
+from apache_beam.io.aws.clients.s3 import messages
+
+try:
+  # pylint: disable=wrong-import-order, wrong-import-position
+  # pylint: disable=ungrouped-imports
+  import boto3
+
+except ImportError:
+  raise ImportError('Missing `boto3` requirement')
```

Comment:
Looking at: https://scans.gradle.com/s/tc45x4bplbo6m/console-log?task=:sdks:python:test-suites:tox:pycommon:docs
Seems like it minds this line. Maybe this is the issue?

```suggestion
  raise ImportError('Missing boto3 requirement')
```